packages/nextjs/app/builders/0x5cc8Be96B1C9A68F57a73b5bEa60cF5D890055A1/page.tsx

```diff
@@ -0,0 +1,85 @@
+import { EnvelopeIcon, GiftIcon, GlobeAltIcon } from "@heroicons/react/24/outline";
+
+// Styles object to keep things organized
+const styles = {
+  container: "min-h-screen p-8 bg-base-200",
+  card: "max-w-2xl mx-auto p-8 bg-base-100 rounded-xl shadow-lg",
+  profileSection: "flex flex-col md:flex-row items-center md:items-start gap-6",
+  avatar: "w-32 h-32 rounded-full border-4 border-base-300",
+  nameSection: "text-center md:text-left",
+  name: "text-3xl font-bold",
+  address: "font-mono text-sm text-base-content/70",
+  section: "mt-8",
+  sectionTitle: "text-xl font-semibold mb-4 text-base-content",
+  bio: "text-base-content/80 leading-relaxed",
+  socialLinks: "flex gap-4 items-center justify-center md:justify-start",
+  socialIcon: "p-2 hover:bg-base-200 rounded-full transition-colors duration-200 w-10 h-10",
+};
+
+const BuilderPage = () => {
+  const builderDetails = {
```

Comment:
You could place the const outside of the component since it is a static profile page.